scaffolds_actor_model/elixir/README.md

```diff
@@ -0,0 +1,34 @@
+# Actor Model
+
+This is an example of actors in Elixir. Since actors are built in
+Erlang/Elixir, this is simply an example of a way to use them.
```

Comment:
I'm actually unsure that `processes` are not already actors, since they have a mailbox. If so, GenServer is *on top of an actor* rather than being an actor. In fact, it's not directly part of Erlang; it's part of OTP, which is considered to be "on top of Erlang".

In short: I'm not sure, so I didn't want to be too specific :s. WDYT ?

Reply:
You're right, I think I saw the same discussion elsewhere (GenServer are not exactly like Actors).